app/src/main/java/org/permanent/permanent/ui/compose/DesignateContactOrStewardScreen.kt

```diff
@@ -0,0 +1,172 @@
+package org.permanent.permanent.ui.compose
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Card
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.Divider
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.core.content.ContextCompat
+import org.permanent.permanent.R
+
+@Composable
+fun DesignateContactOrStewardScreen() {
+    val context = LocalContext.current
+    val scrollState = rememberScrollState()
+
+    val primaryColor = Color(ContextCompat.getColor(context, R.color.colorPrimary))
+    val blackColor = Color(ContextCompat.getColor(context, R.color.black))
+    val lightBlueColor = Color(ContextCompat.getColor(context, R.color.superLightBlue))
+    val whiteColor = Color(ContextCompat.getColor(context, R.color.white))
+    val regularFont = FontFamily(Font(R.font.open_sans_regular_ttf))
+    val boldFont = FontFamily(Font(R.font.open_sans_bold_ttf))
+    val semiBoldFont = FontFamily(Font(R.font.open_sans_semibold_ttf))
+    val smallTextSize = 11.sp
+    val subTitleTextSize = 16.sp
+    val titleTextSize = 19.sp
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(lightBlueColor)
+            .padding(horizontal = 32.dp)
+            .verticalScroll(scrollState),
+        verticalArrangement = Arrangement.Top,
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        Spacer(modifier = Modifier.height(40.dp))
```

Comment:
Instead of spacer can you add a padding vertical of 36 dp even thought it will be a small difference from the design?